packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/UpdatePalletFrozenStatus/SelectPalletFrozenStatus.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo, useState } from 'react'
+import styled from 'styled-components'
+
+import { Select, SelectedOption } from '@/common/components/selects'
+import { TextSmall } from '@/common/components/typography'
+
+import { OptionsPalletFrozenStatus } from './OptionsPalletFrozenStatus'
+import { PalletFrozenStatus } from './UpdatePalletFrozenStatus'
+
+interface Props {
+  selectedStatus?: boolean
+  onChange: (selected: PalletFrozenStatus) => void
+}
+export const SelectPalletFrozenStatus = ({ selectedStatus, onChange }: Props) => {
+  const [, setSearch] = useState('')
+  const selected = useMemo(() => {
+    if (selectedStatus === true) {
+      return 'Enabled'
+    }
+    if (selectedStatus === false) {
+      return 'Disabled'
+    }
+    return selectedStatus
+  }, [selectedStatus])
+  const change = (selected: PalletFrozenStatus, close: () => void) => {
+    onChange(selected)
+    close()
+  }
+  return (
+    <Select
```

Comment:
@dmtrjsg suggested to use a toggle instead [here](https://www.figma.com/file/oD3UTElt3yc4JJfPJv4dn3?node-id=4:20973&mode=design#610789666)